lib/blather/client.rb

```diff
@@ -62,9 +65,7 @@
     client.setup(*ARGV)
   end
 
-  def run(options)
-    $stdin.reopen "/dev/null"
```

Comment:
I don't think I saw an answer to why this line was removed? Was it causing a problem?

Reply:
This code is not just run when demonizing, so it prevented scripts from reading STDIN.

Reply:
I'm concerned this will be a breaking change for anyone who uses Blather as a daemon. Can this be made optional or configurable somehow?

Reply:
I've changed it to reopen when daemonizing from a TTY now.